Dockerfile

```diff
@@ -22,7 +22,45 @@ RUN \
     fontconfig \
     python \
     g++ \
-    build-essential && \
+    build-essential \
```

Comment:
@PierreBesson this would still work only if you use this with this docker file right. In most Jenkins environment you might or might not use docker files I guess.
At least in our setup, we don't deploy the application as Docker containers

Reply:
@deepu105 Yes I can totally see why some people would choose to use docker or not. That's why docker is optional in the ci-cd subgen.
Anyway I'm writing some more docs on how to set up karma with different browsers. How to fix configuration problems with chrome headless. I will also add a section on how to switch to PhantomJS.